core/arch/x86/x86_asm_defines.asm

```diff
@@ -47,18 +47,19 @@
  */
 #ifdef X64
 # ifdef WINDOWS
-#  define NUM_SIMD_SLOTS 6 /* xmm0-5 */
+#  define MCXT_NUM_SIMD_SLOTS 6 /* xmm0-5 */
 # else
-#  define NUM_SIMD_SLOTS 16 /* xmm0-15 */
+#  define MCXT_NUM_SIMD_SLOTS 16 /* xmm0-15 */
 # endif
 # define PRE_XMM_PADDING 16
 #else
-# define NUM_SIMD_SLOTS 8 /* xmm0-7 */
+# define MCXT_NUM_SIMD_SLOTS 8 /* xmm0-7 */
 # define PRE_XMM_PADDING 24
 #endif
-#define XMM_SAVED_REG_SIZE 32 /* for ymm */
+#define YMM_REG_SIZE 32
+#define MCXT_SIMD_SLOT_SIZE YMM_REG_SIZE
 /* xmm0-5/7/15 for PR 264138/i#139/PR 302107 */
-#define XMM_SAVED_SIZE ((NUM_SIMD_SLOTS)*(XMM_SAVED_REG_SIZE))
+#define MCXT_TOTAL_SIMD_SLOTS_SIZE ((MCXT_NUM_SIMD_SLOTS)*(YMM_REG_SIZE))
```

Comment:
This one should be s/YMM_REG_SIZE/MCXT_SIMD_SLOT_SIZE/

Reply:
I had intentionally left YMM_REG_SIZE but I think you're right that's better / more general.

Reply:
Also changed in arch_exports.h